src/components/plan/PlanView.jsx

```diff
@@ -1,123 +1,77 @@
-            console.log(r.data.participants)
-        }).catch((e) => {
-            toast.error('Plan not found')
-            console.log(e.toString())
-            navigate('/malker/')
-        })
+        getPlanData(id)
+            .then((r) => {
+                setTitle(r.data.name);
+                setAuthor(r.data.author);
+                setDate(r.data.date);
+                setLocation(r.data.location);
+                setDescription(r.data.description);
+                setParticipants(r.data.participants);
+                console.log(r.data.participants);
+                setLoading(false);
+            })
+            .catch((e) => {
+                setLoading(false);
+                toast.error("Plan not found");
+                console.log(e.toString());
+                navigate("/malker/");
+            });
     }, [id]);
 
-    useEffect(() => {
-        // Check if the current user is in the list of participants
-        if (participants && uuid) {
-            const currentUserExists = participants.some(participant => participant.uuid === uuid);
-            setJoined(currentUserExists);
-            // console.log(joined, uuid, participants, (!joined))
-        }
-    }, [participants, uuid]);
-
-    const handleJoin = () => {
-        if (!name) {
-            setError("Please fill out all fields")
-            toast.error("Please fill out all fields")
-            return
-        }
-
-        // Call post endpoint
-        addParticipant(id, uuid).then((r) => {
-            toast.success("Join successfull");
-            setJoined(true)
-        }).catch((e) => {
-            toast.error("Can't join into the plan")
-            console.log(e.toString())
-        })
-    }
-
     return (
         <section>
             <BackButton />
-            <DropdownPlan title={title} date={date} location={location} description={description} author={author}/>
-
-            {joined && (
-                <div className="container mx-auto mb-10">
-                    <div
-                        className="grid grid-cols-2 sm:grid-cols-3 md:grid-cols-4 xl:grid-cols-7 gap-10 w-10/12 mx-auto">
-                        {participants.map((user) => (
-                            <UserCard user={user.username} userUuid={user.uuid} currentUserUuid={uuid} key={user}/>
-                        ))}
-                    </div>
+            {loading && (
+                <div className="absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2">
+                    <ProgressBar
+                        visible={true}
+                        height="47"
+                        width="47"
+                        barColor="#0789c2"
+                        borderColor="#3dc2f3"
+                        ariaLabel="progress-bar-loading"
+                        wrapperStyle={{}}
+                        wrapperClass=""
+                    />
                 </div>
             )}
-            {!joined && (
```

Comment:
Why this has been removed?

Reply:
its a bug i think, i just moved it, everything works